client/src/components/Offer/NewTag.tsx

```diff
@@ -0,0 +1,6 @@
+import React from "react";
+import styles from "./Offer.module.scss";
```

Comment:
I think, if you've extracted the NewTag component, you should also extract its styles to its own file.